[Fix] ExampleData, TypeError: unsupported operand type(s) for -: 'numpy.ndarray' and 'Timestamp'

## Summary
`TypeError: unsupported operand type(s) for -: 'numpy.ndarray' and 'Timestamp'` was raised with `ExampleData`.

## Codes
https://github.com/lisphilar/covid19-sir/blob/9aeb277d7cfd1e2c10544f5720b0be49427cb048/tests/test_example_data.py#L18-L40

## Outputs
```
_____________________ TestExampleData.test_one_phase[SIR] ______________________

self = <tests.test_example_data.TestExampleData object at 0x7f6daff9ab80>
model = <class 'covsirphy.ode.sir.SIR'>

    @pytest.mark.parametrize("model", [SIR, SIRD, SIRF, SEWIRF])
    def test_one_phase(self, model):
        example_data = ExampleData()
        example_data.add(model)
        with pytest.raises(ValueError):
            example_data.subset()
        # Cleaned dataset
        clean_df = example_data.cleaned()
        cols = [Term.ISO3, Term.DATE, Term.COUNTRY, Term.PROVINCE, Term.C, Term.CI, Term.F, Term.R, Term.N]
        assert set(clean_df.columns) == set(cols)
        # Subset
        subset_df = example_data.subset(model=model)
        assert subset_df.columns.tolist() == Term.SUB_COLUMNS
        example_data.subset(country=model.NAME)
        example_data.subset_complement(model=model)
        example_data.records(model=model)
        # Specialized
        specialized_df = example_data.specialized(model=model)
        assert specialized_df.columns.tolist() == [Term.DATE, *model.VARIABLES]
        # Non-dimensional
>       nondim_df = example_data.non_dim(model=model)

tests/test_example_data.py:38: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
covsirphy/analysis/example_data.py:172: in non_dim
    df = model.convert(restored_df, tau=self._tau)
covsirphy/ode/sir.py:236: in convert
    df = cls._convert(data, tau)
covsirphy/ode/mbase.py:253: in _convert
    time_series = (df[cls.DATE] - df[cls.DATE].min()).dt.total_seconds() // 60
../.pyenv/3.8/lib/python3.8/site-packages/pandas/core/ops/common.py:70: in new_method
    return method(self, other)
../.pyenv/3.8/lib/python3.8/site-packages/pandas/core/arraylike.py:108: in __sub__
    return self._arith_method(other, operator.sub)
../.pyenv/3.8/lib/python3.8/site-packages/pandas/core/series.py:5636: in _arith_method
    return base.IndexOpsMixin._arith_method(self, other, op)
../.pyenv/3.8/lib/python3.8/site-packages/pandas/core/base.py:1295: in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

left = array([Timestamp('2020-01-23 00:00:00'), Timestamp('2020-01-24 00:00:00'),
       Timestamp('2020-01-25 00:00:00'), Ti...'2020-07-18 00:00:00'),
       Timestamp('2020-07-19 00:00:00'), Timestamp('2020-07-20 00:00:00')],
      dtype=object)
right = Timestamp('2020-01-23 00:00:00'), op = <built-in function sub>

    def arithmetic_op(left: ArrayLike, right: Any, op):
        """
        Evaluate an arithmetic operation `+`, `-`, `*`, `/`, `//`, `%`, `**`, ...
    
        Note: the caller is responsible for ensuring that numpy warnings are
        suppressed (with np.errstate(all="ignore")) if needed.
    
        Parameters
        ----------
        left : np.ndarray or ExtensionArray
        right : object
            Cannot be a DataFrame or Index.  Series is *not* excluded.
        op : {operator.add, operator.sub, ...}
            Or one of the reversed variants from roperator.
    
        Returns
        -------
        ndarray or ExtensionArray
            Or a 2-tuple of these in the case of divmod or rdivmod.
        """
        # NB: We assume that extract_array and ensure_wrapped_if_datetimelike
        #  have already been called on `left` and `right`,
        #  and `maybe_prepare_scalar_for_op` has already been called on `right`
        # We need to special-case datetime64/timedelta64 dtypes (e.g. because numpy
        # casts integer dtypes to timedelta64 when operating with timedelta64 - GH#22390)
    
        if (
            should_extension_dispatch(left, right)
            or isinstance(right, (Timedelta, BaseOffset, Timestamp))
            or right is NaT
        ):
            # Timedelta/Timestamp and other custom scalars are included in the check
            # because numexpr will fail on it, see GH#31457
>           res_values = op(left, right)
E           TypeError: unsupported operand type(s) for -: 'numpy.ndarray' and 'Timestamp'

../.pyenv/3.8/lib/python3.8/site-packages/pandas/core/ops/array_ops.py:216: TypeError
```

## Environment

- CovsirPhy version: 2.22.2
- Python version: 3.8
- Installation: pip
- System: Semaphore CI, Linux

--- a/covsirphy/loading/dataloader.py
+++ b/covsirphy/loading/dataloader.py
@@ -4,6 +4,7 @@
 from datetime import datetime, timezone, timedelta
 from pathlib import Path
 import warnings
+import numpy as np
 import pandas as pd
 from covsirphy.util.argument import find_args
 from covsirphy.util.error import deprecate, DBLockedError, NotDBLockedError, UnExpectedValueError
@@ -431,7 +432,7 @@ class DataLoader(Term):
         self._set_date_location(remote_df)
         remote_df = remote_df.set_index(self._id_cols)
         # Update the current database
-        df = df.replace(0, None).combine_first(remote_df).reset_index().set_index(self._id_cols)
+        df = df.replace(0, np.nan).combine_first(remote_df).reset_index().set_index(self._id_cols)
         # Update citations
         cite_dict = {k: [*v, handler.CITATION] if k in remote_df else v for (k, v) in cite_dict.items()}
         return (df, cite_dict, handler)

--- a/covsirphy/ode/mbase.py
+++ b/covsirphy/ode/mbase.py
@@ -250,7 +250,7 @@ class ModelBase(Term):
             return df.set_index(cls.DATE)
         # Convert to tau-free
         tau = cls._ensure_tau(tau, accept_none=False)
-        time_series = (df[cls.DATE] - df[cls.DATE].min()).dt.total_seconds() // 60
+        time_series = (pd.to_datetime(df[cls.DATE]) - df[cls.DATE].min()).dt.total_seconds() // 60
         df.index = (time_series / tau).astype(np.int64)
         df.index.name = cls.TS
         return df.drop(cls.DATE, axis=1)

